test/e2e_source_map_test.ts

```diff
@@ -22,9 +22,6 @@ import {createOutputRetainingCompilerHost, createSourceReplacingCompilerHost, to
 
 import * as testSupport from './test_support';
```

Comment:
While you're touching this - these tests run very slowly because they parse `lib.d.ts` over and over again. Our other tests reuse the same script file, which speeds them up a lot (from 1s per test to 10ms or so). Could you take a look at the test harness in the other files, and maybe harmonize it and improve on this? Not necessarily in this PR.